tests/containers/validate_btrfs.pm

```diff
@@ -41,7 +44,16 @@ sub _sanity_test_btrfs {
     }
     assert_script_run("echo -e 'FROM $img\\nENV WORLD_VAR Arda' > $dockerfile_path/Dockerfile");
     my $btrfs_head = '/tmp/subvolumes_saved';
-    $rt->info(property => 'Driver', value => 'btrfs');
+
+    if (version->parse(get_docker_version()) >= version->parse('23.0.5')) {
```

Comment:
Is this really because of docker? AFAIK this was changed in libcontainers-common.

Reply:
Oh, could you share a SR for this change? Thanks

Reply:
https://build.opensuse.org/request/show/1072085

Reply:
According to the ticket this only started failing recently though while the SR is two months old...

Reply:
Yup, so it seems like a docker update related issue rather than the SR.